bin/build-helper

```diff
@@ -32,14 +23,47 @@ buildImage() {
 }
 
 pushImages() {
-  createImagesFileIfNotExists
-  cat /dev/null > $BASE_DIR/dist/images.txt
-  if [[ "$REGISTRY" =~ amazonaws.com$ ]]; then
-    eval $(aws ecr get-login --no-include-email)
+  local registry='750663954581.dkr.ecr.us-west-2.amazonaws.com'
+
+  function ecr_check_create_repo {
+    # ECR repos are individual resources - create if not present
+    local repo=$1
+    local policy='{"Version":"2008-10-17","Statement":[{"Sid":"AllowPushPull","Effect":"Allow","Principal":{"AWS":["arn:aws:iam::996426618008:root","arn:aws:iam::907744595866:root","arn:aws:iam::215032187822:root","arn:aws:iam::645691762160:root"]},"Action":["ecr:BatchCheckLayerAvailability","ecr:BatchGetImage","ecr:GetDownloadUrlForLayer"]}]}'
```

Comment:
Don't we want registry to be subject to change?

Reply:
I removed the stuff specific to us.